### DISCUSSION TITLE
How to Fuzzy autocomplete

### DISCUSSION THREAD

**Discussion by captainko** [mfussenegger/nvim-jdtls#267] [Q&A]
Missing WebSocketSession in the completion list.
![image](https://user-images.githubusercontent.com/32269291/169677809-92de6653-3309-4b40-ab8f-d6af61e1f7b4.png)

**Comment by mfussenegger** [comment]
You'll need a completion plugin that supports fuzzy matching.

I think https://github.com/hrsh7th/nvim-cmp support client side fuzzy matching.

Eclipse.jdt.ls does have limited server side fuzzy matching support, but in your example I think it would've filtered the candidate out as well. https://github.com/mfussenegger/nvim-lsp-compl is a plugin that supports server side fuzzy matching.
